### PR TITLE
LPS-146890 translation error success panels

### PR DESCRIPTION
**Pull request by boton**
Issue: https://issues.liferay.com/browse/LPS-146890

On top of #2708

### CODE REVIEW

**Comment by boton** [comment]
ci:test:sf

**Comment by boton** [comment]
ci:test:relevant

**Comment by boton** [comment]
ci:test:translation

**Comment by Hanlf** [comment]
@boton fyi, there are some conflicts for this PR.

File: modules/apps/translation/translation-web/src/main/java/com/liferay/translation/web/internal/display/context/ImportTranslationResultsDisplayContext.java [hunk 1]
```diff
@@ -66,6 +67,26 @@ public int getFailureMessagesCount() {
 		return _failureMessages.size();
 	}
 
+	public String getFailureMessagesCSV(Locale locale) {
+		StringBuilder sb = new StringBuilder();
```
Comment (adolfopa) [inline modules/apps/translation/translation-web/src/main/java/com/liferay/translation/web/internal/display/context/ImportTranslationResultsDisplayContext.java]: Just a note for your future self: in general, we use the `StringBundler` class for concatenation, not `StringBuilder`.

**Review by adolfopa — APPROVED** [review]

**Comment by boton** [comment]
Rebased on top of https://github.com/liferay-lima/liferay-portal/pull/2708 rebased base 😖

**Comment by boton** [comment]
ci:test:sf

**Comment by boton** [comment]
ci:test:relevant

**Comment by boton** [comment]
ci:test:translation

**Comment by boton** [comment]
Hi @liferay-frontend, I need to use the size attribute in the [ContainerFluidTag.java](https://github.com/liferay-lima/liferay-portal/blob/master/modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/ContainerFluidTag.java#L29) but it is hardcoded to "xl".

Can you review if I implemented it https://github.com/liferay-lima/liferay-portal/pull/2728/commits/adf1b8d07ce481568661eb775bc69b134eaee941 correctly?

Thanks in advance!

**Comment by boton** [comment]
dear @qa-lima, I update the copies and the markup 6910fe69b9de32110dd87be54d8ed5271762c98e to fit the new design. I see failing tests related to this update. 

Can you please update the tests to work with this new markup?

Only the `Download CSV Report` button https://github.com/liferay-lima/liferay-portal/pull/2708 is behind a feature flag.

**Comment by Jeremy-Ch** [comment]
Hi @boton According to the test failures in this pr, I have updated the Poshi tests. The new pull request see: https://github.com/liferay-lima/liferay-portal/pull/2739

**Pull request closed without merging**